index.bs

```diff
@@ -1414,11 +1404,13 @@ the {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}} methods.
 </div>
 
 
-### Cryptographic Algorithm Identifier (type {{AlgorithmIdentifier}}) ### {#alg-identifier}
-
-A string or dictionary identifying a cryptographic algorithm and optionally a set of parameters for that algorithm. This type is
-defined in [[!WebCryptoAPI]].
+### Cryptographic Algorithm Identifier (type <dfn>AlgorithmIdentifier</dfn>) ### {#alg-identifier}
 
+A number or string identifying a cryptographic algorithm.
```

Comment:
there needs to be IDL here defining `AlgorithmIdentifier` -- I'm working on that now.